internal/e2e/lds_test.go

```diff
@@ -350,16 +350,11 @@ func TestIngressRouteTLSListener(t *testing.T) {
 	rh.OnDelete(s1)
 	assertEqual(t, &v2.DiscoveryResponse{
 		VersionInfo: "0",
-		Resources: []types.Any{
```

Comment:
I had to remove this validation with the changes I did. Is this an expected change? Have I made a mistake by removing this as when the secret is deleted?

Reply:
I don't think you should have to remove this with the changes you made. At least this comparison is looking at the listener object. Double check that if statement that I commented on previously, it might be causing this error.

Reply:
Yeah, we can't remove this test, if it no longer passes then we need to dig into the change made in internal/dag and write a test to locate where the mistake is.